Ports/ruby/package.sh

```diff
@@ -18,7 +18,19 @@ launcher_command="/usr/local/bin/ruby /usr/local/bin/irb --legacy"
 launcher_run_in_terminal="true"
 icon_file="../ruby-kit/ruby.png"
 
-configopts=("--with-coroutine=x86" "--disable-install-doc")
+configopts=("--disable-install-doc")
```

Comment:
```suggestion
configopts=(
    "--disable-install-doc"
)
```

If we are already touching it, we might as well bring it in-line with the (envisioned) style for ports scripts.